core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -156,6 +164,18 @@ public int defineWayBits(int index, int shift) {
         carBwdMaxspeedEncoder = new EncodedDoubleValue("car bwd maxspeed", shift, speedBits, speedFactor, 0, maxPossibleSpeed, true);
         shift += carBwdMaxspeedEncoder.getBits();
 
+        /* Value range: [3.0m, 5.4m] */
```

Comment:
Here (maybe in a later version) we could do kind some kind of a compression: i.e. assign 

 * 2.5m to index 0
 * 3.0m to index 1
 * 3.1m to index 2

... so we could make it more precise with less bits and for the ones that do not fit we would still round. For this taginfo is useful: http://taginfo.openstreetmap.org/keys/height#values ... it seems 2m is very important for the world wide case. (Or even more sophisticated and area dependent and create statistics in the first parse run and then pick the best compression automatically)